simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/LocationFragment.kt

```diff
@@ -292,7 +292,7 @@ class LocationFragment : Fragment(), LocationListener {
     val address: Address?
 
     try {
-      address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)[0]
+      address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults)!![0]
```

Comment:
Same as above, might be more readable like `address = geocoder.getFromLocation(location.latitude, location.longitude, maxResults).first()`